apps/netlify/frontend/src/app/edit-site-modal.js

```diff
@@ -44,17 +47,29 @@ export const EditSiteModal = ({
   const [siteId, setSiteId] = useState(PICK_OPTION_VALUE);
   const [displayName, setDisplayName] = useState('');
   const [isDeploysOn, setIsDeploysOn] = useState(false);
+  const [isAssetDeploysOn, setIsAssetDeploysOn] = useState(false);
   const [availableContentTypes, setAvailableContentTypes] = useState([]);
+  const [filteredContentTypes, setFilteredContentTypes] = useState([]);
   const [selectedContentTypes, setSelectedContentTypes] = useState([]);
-  const [contentTypeQuery, setContentTypeQuery] = useState('');
+  const [isSelectedContentTypesInvalid, setIsSelectedContentTypesInvalid] = useState(false);
 
   const isNewSite = configIndex === undefined || configIndex === null;
 
   const selectId = `site-select-${configIndex ?? 'new'}`;
   const inputId = `site-input-${configIndex ?? 'new'}`;
   const deploysId = `deploys-checkbox-${configIndex ?? 'new'}`;
+  const assetDeploysId = `asset-deploys-checkbox-${configIndex ?? 'new'}`;
   const contentTypeSelectId = `content-type-select-${configIndex ?? 'new'}`;
 
+  const availableNetlifySites = useMemo(() => {
```

Comment:
It looks like the computation that's being done is very simple, it doesn't feel like we need memoize this. Were you seeing performance problems?